concordium-std/src/types.rs

```diff
@@ -1045,6 +1045,18 @@ impl FromStr for PublicKeyEd25519 {
     }
 }
 
+#[cfg(feature = "concordium-quickcheck")]
+impl quickcheck::Arbitrary for PublicKeyEd25519 {
+    fn arbitrary(g: &mut quickcheck::Gen) -> Self {
+        let lower: u128 = quickcheck::Arbitrary::arbitrary(g);
+        let upper: u128 = quickcheck::Arbitrary::arbitrary(g);
+        let mut out = [0u8; 32];
+        out[..16].copy_from_slice(&lower.to_le_bytes());
+        out[16..].copy_from_slice(&upper.to_le_bytes());
```

Comment:
So I know I added this, and it is "OK" for the purposes here, but technically this also generates invalid public keys since not all 32-byte arrays are valid public keys. Can you please document this here.

Reply:
Done.